src/iam/iam-base.ts

```diff
@@ -340,7 +357,7 @@ export class IAMBase {
                 break;
             }
             default:
-                throw new Error("Event not supported");
+            // throw new Error("Event not supported");
```

Comment:
Why this change?

Reply:
Ah, checking something and I committed this accidentally. Reverted.